package.json

```diff
@@ -14,6 +14,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "axios": "^0.17.1",
```

Comment:
since node 8, axios isn't needed anymore
you can just use `fetch`: `const response = await fetch(url, options);`

Reply:
Are you referring to the node-fetch library?

Reply:
https://github.com/matthew-andrews/isomorphic-fetch

Reply:
Guess I merged this PR too soon :sweat_smile: 

Anyway, this is fixed on #4.